common/scripts/lib/_helpers.sh

```diff
@@ -506,6 +506,56 @@ __copy_script_remote() {
   copy_cmd_out=$(eval $copy_cmd)
 }
 
+# accepts arguments $host $port $serviceName $timeout
+# $timeout is optional and defaults to 60 seconds if not defined
+__check_service_connection() {
+
+  if [ "$#" != "3" ] && [ "$#" != "4" ]; then
+    echo "invalid arguments to __check_service_connection()"
```

Comment:
why are we not using __process_error and __process_msg here like the rest of the file?

Reply:
@ric03uec 
tried using that but it was giving error - 
```
47ff0ededb29b920037c3070ef8eb85d305193e3d73109261d2add2023e31c24
|___ Checking db container status on: localhost:5432
/home/niranjan/shippable/admiral/common/scripts/lib/_helpers.sh: line 548: process_msg: command not found
```

should i source `logger.sh` in `helper.sh`?